libs/pipingsidesheet/src/lib/utils-sidesheet/useElectricalNetwork.ts

```diff
@@ -0,0 +1,41 @@
+import { useHttpClient } from '@equinor/fusion-framework-react-app/http';
+import { ElectricalNetwork, useContextId } from '@cc-components/shared';
+
+import { useQuery } from '@tanstack/react-query';
+
+export const useElectricalNetworks = (facility: string, tagNos: string[]) => {
+  const client = useHttpClient('electrical-api');
+  const contextId = useContextId();
+
+  const { data, isLoading, error } = useQuery<ElectricalNetwork[], Error>(
+    [facility, tagNos],
```

Comment:
I would add a word identifying the resource more than just facility, tagnos. Like i.e electrical-networks, facility, tagnos. Makes it way easier when debugging or invalidating